apps/engmanagement/src/pages/api/auth/[...nextauth].ts

```diff
@@ -12,7 +12,7 @@ export const nextAuthOptions: NextAuthOptions = {
   session: {
     strategy: 'jwt',
   },
-  adapter: PrismaAdapter(prisma as any),
+  adapter: PrismaAdapter(prisma),
```

Comment:
`any` is no longer required since everything uses the same prisma client source from teh database packages so types are in alignment